src/llvm/type.cr

```diff
@@ -173,7 +173,7 @@ struct LLVM::Type
     Value.new LibLLVM.const_array(self, (values.to_unsafe.as(LibLLVM::ValueRef*)), values.size)
   end
 
-  def inline_asm(asm_string, constraints, has_side_effects = false, is_align_stack = false, can_throw = false)
+  def inline_asm(asm_string, constraints, has_side_effects = false, is_align_stack = false, intel_dialect = false, can_throw = false)
```

Comment:
**question:** This is technically a breaking change because parameter positions are shifted. Yould we add `intel_dialect` as a non-positional parameter (or as last positional parameter)?
```suggestion
  def inline_asm(asm_string, constraints, has_side_effects = false, is_align_stack = false, can_throw = false, *, intel_dialect = false)
```